executor/simple.go

```diff
@@ -948,11 +948,17 @@ func (e *SimpleExec) executeCreateUser(ctx context.Context, s *ast.CreateUserStm
 
 	var userAttributes any = nil
 	if s.CommentOrAttributeOption != nil {
-		if s.CommentOrAttributeOption.Type == ast.UserCommentType {
-			userAttributes = fmt.Sprintf("{\"metadata\": {\"comment\": \"%s\"}}", s.CommentOrAttributeOption.Value)
-		} else if s.CommentOrAttributeOption.Type == ast.UserAttributeType {
-			userAttributes = fmt.Sprintf("{\"metadata\": %s}", s.CommentOrAttributeOption.Value)
+		if s.CommentOrAttributeOption.Type != ast.UserResourceGroupName {
+			if s.CommentOrAttributeOption.Type == ast.UserCommentType {
+				userAttributes = fmt.Sprintf(`{"metadata": {"resource_group": "default", "comment": "%s"}}`, s.CommentOrAttributeOption.Value)
```

Comment:
Since user can also alter `resource_group` by `alter user ... attribute ...`, is it better to extract `resource_group` outside `metadata` like:
```json
{
	"metadata":  {"comment": "...."}
	"resource_group": "default",
}
```

Reply:
I made such a json, is it ok?
{"metadata": {"comment": "....", "resource_group": "default"}}

Reply:
fixed.